css/style.css

```diff
@@ -217,35 +226,119 @@ q::after {
     margin: 0;
     padding: 100px 50px 50px;
   }
-}
 
-/**
- ********************
- *
- *  height: auto 
-   no height, adaptable 
+  /*  ____________  store page and form   ___________________ */
 
+  .form_wrapper {
+    width: 100%;
+    display: flex;
+  }
 
-   max height 
+  .form {
+    width: 50%;
+  }
 
+  .form_poster {
+    width: 50%;
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
 
+  .form_field {
+    width: 75%;
+    margin: 20% auto;
+    text-align: left;
+    border: 0;
+  }
```

Comment:
I'm not really sure how to get proportions right, like in the case of fieldset size compering with other elements on the size, and figma is only somewhat helpful.